tasks/main.yml

```diff
@@ -38,3 +40,16 @@
     state: present
   environment:
     - ACCEPT_EULA: Y
+
+- name: Gather the package facts
+  ansible.builtin.package_facts:
```

Comment:
for now, use the old-style module `package_facts` instead of the fully qualified collection name (FQCN) `ansible.builtin.package_facts`